canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -87,6 +88,20 @@ class DropDown extends React.Component {
 		return selectedOption;
 	}
 
+	getItemIcon(propertyId, enumValue) {
```

Comment:
`propertyId` is never used and `enumValue` is actually passed into the function below.